experimental/origin-dapp2/src/pages/growth/Invite.js

```diff
@@ -0,0 +1,252 @@
+                onClick={this.handleCopyClick}
+              >
+                {showCopyConfirmation && (
+                  <Fragment>
+                    <img src="/images/growth/checkmark.svg" />
+                    <div className="ml-2">Copied</div>
+                  </Fragment>
+                )}
+                {!showCopyConfirmation && <div>Copy</div>}
+              </div>
+            </div>
+          </div>
+          <div className="col-4 pl-4 pr-0">
+            <div className="normal">Share or Tweet</div>
+            <div className="d-flex mt-2">
+              <button
+                className="social-btn fb"
+                onClick={this.handleFbShareClick}
+              >
+                <img src="/images/growth/facebook-icon.svg" />
+              </button>
+              <button className="social-btn tw">
+                <img src="/images/growth/twitter-icon.svg" />
+              </button>
+            </div>
+          </div>
+        </div>
+
+        <div className="empasis mt-5">Invite via Email</div>
+        <div>Enter email addresses of friends you want to invite</div>
+        <textarea
+          name="invite-email"
+          className="email-text p-3"
+          cols="50"
+          rows="5"
+          placeholder="Separate email addresses with commas."
+        />
+        <button
+          className="btn btn-primary btn-rounded mt-2"
+          children="Invite Friends"
+        />
+      </div>
+    )
+  }
+
+  renderTrackInvites() {
+    return <div />
+  }
+
+  render() {
+    const { subPage } = this.state
+    return (
+      <div className="container growth-invite">
+        <div>
+          <Link to="/campaigns" className="back d-flex mr-auto">
+            <img src="/images/caret-blue.svg" />
+            <div>Back to Campaign</div>
+          </Link>
+          <h1 className="mb-2 pt-3 mt-3">Invite your friends to Origin</h1>
+          <div>Get Origin Tokens by completing the tasks below.</div>
+        </div>
+
+        <div className="navigation-list d-flex justify-content-left mt-4">
+          <NavigationItem
+            selected={subPage === 'sendInvites'}
+            onClick={() => this.handleNavigationClick('sendInvites')}
+            title={`Send Invites`.toUpperCase()}
+          />
+          <NavigationItem
+            selected={subPage === 'trackInvites'}
+            onClick={() => this.handleNavigationClick('trackInvites')}
+            title={`Track Invites`.toUpperCase()}
+          />
+        </div>
+        {subPage === 'sendInvites' && this.renderSendInvites()}
+        {subPage === 'trackInvites' && this.renderTrackInvites()}
+      </div>
+    )
+  }
+}
+
+export default GrowthInvite
+
+require('react-styl')(`
+  .growth-invite.container
+    max-width: 760px;
+  .growth-invite
+    margin-top: 70px;
+    .back img
+      width: 15px;
+      margin-right: 6px;
+      -webkit-transform: rotate(270deg);
```

Comment:
no need to add vendor prefixed styles - should get added automatically when the css is built

Reply:
Also no need for semi colons but I guess it's a pain to remove them if pasting styles in :-)

Reply:
Nice, didn't notice when css compiles id adds vendor specific styles, thanks. Also removed all semicolons from growth components